chainer/links/connection/bias.py

```diff
@@ -0,0 +1,57 @@
+from chainer.functions.math import bias
+from chainer import link
+
+
+class Bias(link.Link):
+    """Broadcasted elementwise summation with learnable parameter.
+
+    Computes a elementwise summation as :func:`~chainer.functions.bias`
+    function except that its second input is a learnable bias parameter
+    :math:`b` the link has.
+
+    Args:
+        axis (int): The first axis of the first input of
+            :func:`~chainer.functions.bias` function along which to apply its
+            second input.
+        shape (tuple of ints): Shape of the learnable bias parameter. If
+            ``None``, a bias parameter needs to be given explicitly to its
+            ``__call__`` method's second input.
+
+    .. seealso:: See :func:`chainer.functions.bias` for details.
+
+    Attributes:
+        b (~chainer.Variable): Bias parameter.
+
+    """
+    def __init__(self, axis=1, shape=None):
+        super(Bias, self).__init__()
+
+        # Add b parameter if given.
+        if shape is not None:
+            self.add_param('b', shape)
+            self.b.data.fill(0)
+
+        # Hold axis.
```

Comment:
I think this comment is not needed as we can understand it from the following line.